fearless/Common/Network/Wallet/WalletNetworkOperationFactory+Protocol.swift

```diff
@@ -51,7 +51,8 @@ extension WalletNetworkOperationFactory: WalletNetworkOperationFactoryProtocol {
         let compoundReceiver = createAccountInfoFetchOperation(receiver)
 
         let builderClosure: ExtrinsicBuilderClosure = { builder in
-            let call = SubstrateCallFactory().transfer(to: receiver, amount: amount)
+            // NOT USED
```

Comment:
??

Reply:
code from common wallet (capital). need to remove all capital-based code from app